buildSrc/src/main/java/com/google/firebase/gradle/plugins/ReleaseGenerator.kt

```diff
@@ -14,7 +14,7 @@
 package com.google.firebase.gradle.plugins
 
 import com.google.common.collect.ImmutableList
-import java.io.File
+import com.google.gson.Gson
```

Comment:
tbh- I would opt for `kotlinx.serialization`, since we [already use that in other places.](https://github.com/firebase/firebase-android-sdk/blob/master/buildSrc/src/main/java/com/google/firebase/gradle/plugins/ReleaseConfig.kt#L35)